plutus-tx-plugin/test/Plugin/Data/Spec.hs

```diff
@@ -50,31 +50,31 @@ monoData = testNested "monomorphic" [
   , goldenPir "synonym" synonym
   ]
 
-data MyEnum = Enum1 | Enum2
+data MyEnum = EnumA1 | EnumB2
```

Comment:
You don't need any of these changes any more, and I think we shouldn't have them. I actually want to test that the previous version works.

The name clash checker should be part of the parser, remember, so I want to know that if I generate a program with name clashes but good uniques, it works.